tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -562,6 +581,11 @@ public bool DiagnosticSourceEnabled
         /// </summary>
         internal bool IsRunningInAzureAppService { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the tracer should propagate service data in db queries
+        /// </summary>
+        internal string DbmPropagationMode { get; set; }
```

Comment:
If we have an enum, I'd stick with the enum and not convert to string afterwards. Having an enum can sometime be tricky for backward compatibility, but in settings it's ok.